src/ui/components/DashKit/plugins/GroupControl/Control/store/types.ts

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+
+import {ResponseSuccessControls} from 'ui/libs/DatalensChartkit/modules/data-provider/charts/types';
+
+import {ErrorData, LoadStatus} from '../../../Control/types';
+
+export type Dispatch = React.Dispatch<Action>;
+
+export type State = {
+    status: LoadStatus;
+    loadedData: null | ResponseSuccessControls;
+    errorData: null | ErrorData;
+    loadingItems: boolean;
+    validationError: null | string;
+    isInit: boolean;
+};
+
+type SetErrorData = {
+    type: 'SET_ERROR_DATA';
+    payload: {status: LoadStatus; errorData: ErrorData};
+};
+export const setErrorData = (payload: SetErrorData['payload']): SetErrorData => {
+    return {
+        type: 'SET_ERROR_DATA',
```

Comment:
Due to our agreements we decided to use Symbol and scope for action names, for ex.
Symbol('dash/SET_CONTROL_ERROR_DATA')